CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fix Vulnerability Detector E2E tests by adding description to all tests([#5151](https://github.com/wazuh/wazuh-qa/pull/5151)) \- (Tests)
```

Comment:
```suggestion
- Fix Vulnerability Detector E2E tests by adding description to all tests ([#5151](https://github.com/wazuh/wazuh-qa/pull/5151)) \- (Tests)
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/5151/commits/eea576b2737cb1afb0a13b8dd09801781e6defaf